### ISSUE TITLE
`available_api_versions` broken url

### ISSUE THREAD

**Issue by eimrek** [aiidateam/aiida-optimade#381]
See e.g. https://aiida.materialscloud.org/mc3d/optimade/v1/info

```
      "available_api_versions": [
        {
          "url": "https://aiida.materialscloud.org/v1",
          "version": "1.1.0"
        }
      ],
```

https://aiida.materialscloud.org/v1 points to nothing.

Reported by @blokhin

**Comment by blokhin** [comment]
@giovannipizzi @CasperWA @unkcpz please FIXME

**Comment by unkcpz** [comment]
I think the problem is here https://github.com/aiidateam/aiida-optimade/blob/e8275462e40b8533e76ea13e5bf58edd9858f88c/aiida_optimade/routers/info.py#L44
It should be `root_path` rather than the base_url. @eimrek let's try to create a branch and redeploy again together~

**Comment by unkcpz** [comment]
fixed by #471

**Comment by blokhin** [comment]
OK, I herewith confirm the Materials.Cloud databases [started to work](https://optimade.science/?filter=nelements>0&providers=mc3d,mc2d) in Optimade.Science (without any fixes from my side).
